<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>459</y>
      <width>801</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFF;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>521</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>471</width>
        <height>51</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="correctButton">
       <property name="text">
        <string>Correct</string>
       </property>
      </widget>
      <widget class="QPushButton" name="nearButton">
       <property name="text">
        <string>Close/Near</string>
       </property>
      </widget>
      <widget class="QPushButton" name="incorrectButton">
       <property name="text">
        <string>Incorrect</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;</string>
    </property>
    <property name="title">
     <string>Image Results</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 2em;</string>
     </property>
     <property name="text">
      <string>Original Image</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 2em;</string>
     </property>
     <property name="text">
      <string>OD Detected Image</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QGraphicsView" name="original">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>361</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #eee;</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="preprocessed">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>361</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #eee;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
